src/routes/member/[studentId]/+page.server.ts

```diff
@@ -0,0 +1,114 @@
+import prisma from "$lib/utils/prisma";
+import { error, fail } from "@sveltejs/kit";
+import type { PageServerLoad } from "./$types";
+import { ctxAccessGuard } from "$lib/utils/access";
+import apiNames from "$lib/utils/apiNames";
+import { Prisma } from "@prisma/client";
+import { _classProgrammes } from "./data";
+
+export const load: PageServerLoad = async ({ params }) => {
+  const [member, publishedArticles] = await Promise.all([
```

Comment:
Apparently you're [supposed to avoid Promise.all](https://chrysanthos.xyz/article/dont-ever-use-promise-all/)

Reply:
okidoki